documentation/modules/mtv-performance-recommendation.adoc

```diff
@@ -0,0 +1,174 @@
+
+.Example
+
+* 80 single disk VMs, containing 35 GiB of data each, using a single host took 2 hours and 43 minutes, with a migration rate of 294 MiB/s.
+
+* 80 single disk VMs, containing 35 GiB of data each, using 8 ESXi hosts took 41 minutes, with a migration rate of 1,173 MiB/s.
+
+[NOTE]
+====
+From the aforementioned examples, it is evident that migrating 80 VMs from 8 ESXi hosts, 10 from each host, concurrently is four times faster than running the same VMs from a single ESXi host. 
+
+Migrating a larger number of VMs from more than 8 ESXi hosts concurrently could potentially show increased performance. However, it was not tested and therefore not recommended.
+====
+
+[id="mtv-performance-bios-profile_{context}"]
+== Set ESXi hosts BIOS profile and ESXi Host Power Management for High Performance
+
+Where possible, ensure that hosts used to perform migrations are set with BIOS' profiles related to maximum performance.  Hosts which use Host Power Management controlled within vSphere should check that ‘High Performance’ is set.
+
+Testing showed that when transferring more than 10 VMs with both BIOS and host power management set accordingly, migrations had an increase of 15 MiB in the average datastore read rate.
+
+[id="mtv-performance-multiple-migration-plans_{context}"]
+== Multiple migration plans compared to a single large migration plan
+
+The maximum number of disks that can be referenced by a single migration plan is 500. For more details, see link:https://issues.redhat.com/browse/MTV-1203[(MTV-1203)]. 
+
+When attempting to migrate many VMs in a single migration plan, it can take some time for all migrations to start.  By breaking up one migration plan into several migration plans, it is possible to start them at the same time.
+
+Comparing migrations of:
+
+* 500 VMs using 8 ESXi hosts in 1 plan, `max_vm_inflight=100`,  took 5 hours and 10 minutes.
+
+* 800 VMs using 8 ESXi hosts with 8 plans, `max_vm_inflight=100`, took 57 minutes.
+
+Testing showed that by breaking one single large plan into multiple moderately sized plans, for example, 100 VMS per plan, the total migration time can be reduced.
+
+[id="mtv-performance-max-values_{context}"]
+== Maximum values tested
+
+* Maximum number of ESXi hosts tested: 8
+* Maximum number of VMs in a single migration plan: 500
+* Maximum number of VMs migrated in a single test: 5000
+* Maximum number of migration plans performed concurrently: 40
+* Maximum single disk size migrated: 6 T disks, which contained 3 Tb of data
+* Maximum number of disks on a single VM migrated: 50
+* Highest observed single datastore read rate from a single ESXi server:  312 MiB/second
+* Highest observed multi-datastore read rate using eight ESXi servers and two datastores: 1,242 MiB/second
+* Highest observed virtual NIC transfer rate to an {ocp-name} worker: 327 MiB/second
+* Maximum migration transfer rate of a single disk: 162 MiB/second (rate observed when transferring 1.5Tb case/time using utilized data)
```

Comment:
rate observed when transferring 1.5Tb case/time using utilized data - ether remove ether change to "nonconcurrent migration of  1.5Tb utilized data)